### DISCUSSION TITLE
Payment providers for Openmage (long term solution)

### DISCUSSION THREAD

**Discussion by DegrizNet** [OpenMage/magento-lts#3339] [Extensions & Modules]
Hello,

I am a big fan of Openmage and have used it to create over a hundred online shops. I find it particularly suitable for medium to smaller projects compared to Magento 2. However, I have recently noticed a potential long-term issue with payment modules. It seems that most payment providers would not be willing to maintain their modules for Openmage, which could be a significant problem. The availability of payment options is crucial for making Openmage attractive to users. Currently, I can only see one listed payment provider on the official Openmage website.

In my opinion, there are a few possible solutions to address this issue. One approach is to convince major payment providers that there is still a demand for Openmage and encourage them to continue supporting it. Another option could be to seek donations or sponsorships to finance the development of important modules. Alternatively, it might be helpful to have an updated list of developers who are actively working on Openmage modules. This could be achieved through a marketplace or directory listing where developers can showcase and share their contributions.

Here are a couple of examples related to payment modules for Magento 1 (now archived):
https://github.com/stripe-archive/stripe-magento1-releases
https://docs.klarna.com/platform-solutions/adobe-commerce/magento-1x/magento-1x/

**Comment by DegrizNet** [comment]
I've been thinking, and perhaps it would be good for the Openmage organization and its developers to have some income from their work. One option could be something like Openmage Plus, where customers who pay a monthly fee would get the opportunity to have certain features developed on a priority basis, receive assistance and advice, and maybe even access to additional modules.

I have invested a lot in developing modules for Magento, and I have already developed over 100 modules for my clients. I don't sell these modules individually, as they are customized for my clients. However, considering that if there is no support from payment gateway providers etc., many customers may switch to another system, and consequently, I would have to work on a different platform. Therefore, I am already contemplating whether it would be better for me to switch to another platform like Shopify,Woocomerce,... 
I believe Magento 2 might be an overkill for the majority of my clients.

Also most of Magento 1 users do not know anything about Openmage and how great it is. ;)

**Comment by fballiano** [comment]
I was about to post something along the lines of your message :-)

I do not really like subscription services like an "openmage plus" but in my opinion we should have a "sponsor task development" published somewhere, with task ideas that we may want to implement and the way for somebody to pay for that.

I'm not really sure if that would really be used by somebody tho, because I think most of the openmage stores already have some developer or they reach to us in case they need something.

**Reply by DegrizNet** [reply]
I think "sponsor task development" this is a good way to start and see what is the response on that.

There should be some income coming to Openmage to promote platform, get new partners and developers etc.

**Reply by elidrissidev** [reply]
> in my opinion we should have a "sponsor task development" published somewhere, with task ideas that we may want to implement and the way for somebody to pay for that.

Something similar was in #1785

**Reply by DegrizNet** [reply]
Ok before this conversion becomes obsolete if we talk about Stripe. How can we ensure that Stripe payment is officially supported for Openmage? I know that the Stripe module for Magento 1.9 is still functional, but it will require future updates. The best approach, in my opinion, would be to contact Stripe and request their support for Openmage. However, if that is not possible, what should we do? How much sponsorship would be needed for maintenance?
